data/mil2525d/utilities/military-features-utilities/template-gdb-toolbox/scripts/CreateTemplateGDB/UpdateDomains.py

```diff
@@ -0,0 +1,59 @@
+# ----------------------------------------------------------------------------------
+# Copyright 2015 Esri
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+# UpdateDomains.py
+#
+# Description: Updates the domains in a specified military features geodatabase,
+# using the domain CSV files found in the specified folder.
+#
+# Requirements: ArcGIS Desktop
+# ----------------------------------------------------------------------------------
+import arcpy, os, sys, traceback
+import glob
+
+from arcpy import env
+from os import path
+
+def updateDomains(domainsFolder, targetGDB):
```

Comment:
This is much, much better&simpler than that crazy model we had - nice job & thanks for simplifying this